[2025-09-15T21:15:00.732+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-15T21:15:00.840+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T19:00:00+00:00 [queued]>
[2025-09-15T21:15:00.859+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T19:00:00+00:00 [queued]>
[2025-09-15T21:15:00.860+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-15T21:15:00.891+0200] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-09-15 19:00:00+00:00
[2025-09-15T21:15:00.910+0200] {standard_task_runner.py:64} INFO - Started process 1249 to run task
[2025-09-15T21:15:00.922+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'extract_customers', 'scheduled__2025-09-15T19:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmpam7ubwa1']
[2025-09-15T21:15:00.928+0200] {standard_task_runner.py:91} INFO - Job 35: Subtask extract_customers
[2025-09-15T21:15:01.104+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T19:00:00+00:00 [running]> on host f51a750ef2c3
[2025-09-15T21:14:59.367+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-15T19:00:00+00:00'
[2025-09-15T21:14:59.369+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-15T21:14:59.543+0200] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-15T21:14:59.544+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-15T21:14:59.593+0200] {python.py:237} INFO - Done. Returned value was: 1
[2025-09-15T21:14:59.596+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-15T21:14:59.648+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=extract_customers, run_id=scheduled__2025-09-15T19:00:00+00:00, execution_date=20250915T190000, start_date=20250915T191500, end_date=20250915T191459
[2025-09-15T21:14:59.687+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-15T21:14:59.741+0200] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-15T21:14:59.745+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
